.github/PULL_REQUEST_TEMPLATE

```diff
@@ -6,27 +6,42 @@ Fixes JIRA ticket xxx
 
 ## How has this been tested?
 
-<!--- Please describe in detail how you tested your changes. -->
+<!-- Please describe in detail how you tested your changes. -->
+
+-
 
 ## Accessibility concerns or updates
 
-<!--- Describe any accessibility concerns or updates that were made that should be known. -->
+<!-- Describe any accessibility concerns or updates that were made that should be known. -->
+
+-
+
+### Accessibility Checklist
```

Comment:
Add a checkbox about color contrast.

Reply:
I like what you added.  That's what I assumed would be the approach for checking the color contrast.

Reply:
I'd also prefer to keep devs in Storybook for a11y checking.